__device-tests__/utils.js

```diff
@@ -40,8 +40,7 @@ const SUPPORTED_GLOBAL_STYLES_ELEMENTS = [
  * @return {Buffer} Sreenshot image.
  */
 export async function takeScreenshotByElement( element, { padding } = {} ) {
-	const sessionCapabilities = await editorPage.driver.sessionCapabilities();
```

Comment:
From my research, WebdriverIO provides a [`getDisplayDensity`](https://webdriver.io/docs/api/appium/#getdisplaydensity) method for Android, but no equivalent for iOS, which is unfortunately the platform on which we use/need pixel ratio.

This implementation instead relies upon a static capability configuration set in the [sibling PR](https://github.com/WordPress/gutenberg/pull/55322/files#diff-14b5427ac3a466e50e4f22aa9b8e218a07cbdc55662effc4bf57b59366b91e40R14).

Reply:
I've just stumble on an issue on Android where it is expecting the pixel ratio [see these lines](https://github.com/wordpress-mobile/gutenberg-mobile/pull/6281/files#diff-17df8429cf2b4cca490b47704148d667ae52f41958836ab5a82fb243e69db41eR67-R71). If we update this to:

`const pixelRatio = editorPage.driver.capabilities.pixelRatio` it will work for both cases (if we update the iOS capabilities in the sibling PR to use `pixelRatio` as the name of the fixed values)

Reply:
Good catch. I interpret this to mean implementing your [prior feedback](https://github.com/WordPress/gutenberg/pull/55322#discussion_r1360399049) will address this issue. I am implementing that now. 👍🏻

Reply:
Thank you!